# Research Work and Interests

## Current Research 

**Computational analysis of next-generation sequencing data in squamous cell oncogenesis.**<br>
**Sponsor:** Professor Nicholas Coleman<br>
**Collaborators:** Professor Eugene Healy (Univ. Southampton), Prof. Chris Smith (Cambridge, Biochemistry) & Dr. Anton Enright (Univ. Cambridge/EMBL-EBI)<br>

Squamous cell carcinogenesis is a complex, highly integrated systems-based process. In order to more fully understand the nature of genomic processes underlying the transformation from epithelium to SCC I am interrogating large scale NGS datasets (both RNA-seq and DNA-seq). This MRC funded post-doctoral fellowship is focused on the analysis and integration of expression profiling and genomic sequencing data, including network analysis and comparative analysis of publically available data (such as from TCGA) with our experimental work. The project is particularly trageted at understanding the role of alternative RNA splicing in the pathogenesis of squamous cell carcinoma. We will use patient-derived samples from skin and cervix and aim to translate the findings across squamous-type carcinomas in other organ systems.

**Understanding transcriptional events in melanoma carcinogenesis for early diagnosis, prognosis and novel therapeutic approaches.**<br>  

Leveraging extensive and detailed publically available data collections of next-generation sequencing and microarray analyses of melanoma samples I have applied machine learning and network biology analyses to understand the early events that transition primary melanoma to metastatic disease. In order to further understand how molecular events drive melanoma initiation and progression I am leading a project targeting co-dependent pathways in order to identify genes showing genetic dosage lethality in melanoma. The project aims to translate these findings into potential avenues for therapy using repositioned existing drug treatments.

**Drug design and repositionaing**<br>

The development of new theraies is costly and time-consuming. There exists an enormous library of therapeutic compunds whose activity can be predicted to extend across multiple diseases, but which are not used in such a way. By analysing transcriptomic and genetic data from diseases of interest (atopic drmatitis, skin cancer, cervical cancer etc) we aim to identify promising compunds with established pharmacological profiles to test in novel therapeutic areas, shortening the time from discovery to prescription.

**Non-invasive Diagnostics in Skin Disease**<br>

The gold standard non-invasive diagnostic tool for skin cancer by dermatologists is the dermatoscope. Presently dermatoscopes are expensive and affordable models lack digital capabilities. These factors make their use outside specilist centres in wealthy countries infeasible. I am directing a project aiming to produce and refine a low-cost, simple digital dermatoscope for widespread use in clinical contexts where cost sensitivity curently limits their deployment. Additional imaging capabilities will also be integrated to the device to enable it to perform as a cutting-edge research tool in the optical investigation of skin biology. Future work will be focused on leveraging this tool as a platformm for computationally aided diagnostics with important potential applications in resource-limited settings. This work has been supported by a grant from the University of Cambridge Biomaker Challenge and will be published as an open-source hardware project.

## Collaborations

**Prof. Jacquline Boultwood** *(University of Oxford)*. RNA-seq and splicing analysis in myelodysplastic syndrome (Pellagetti et al, Blood 2018)<br>
**Narita lab** *(University of Cambridge Cancer Institute)*. Molecular basis of sensescence and its role in carcinogenesis (Parry et al, Nature Communications 2018).<br>
**Coleman Lab** *(University of Cambridge)*. Ongoing collaboration with Coleman group members on bioinformatics aspects of work in squamous cell carcinoma, HPV biology and paediatric tumours. (Kucia-Tran et al, British Journal of Cancer 2016, Journal of Pathology 2018) <br>
**Raffan/O'Rahilly group** *(University of Cambridge, IMS)*. Collaboration on data analysis of genetic effects on obesity in dogs. (Raffan et al, Cell Metabolism 2016, PeerJ 2015).

## Previous Research

### Masters Thesis 

**Functional genomics analysis of papillomavirus oncogenesis in W12 cell lines**<br>  
**Supervisor:** Professor Nicholas Coleman<br>  
This research explored the functional genomic aspects of HPV carcinogenesis applying network models to high-quality gene expression data in a cell model of HPV16 mediated oncogenesis (Smith et al, Scientific Reports 2016).

### PhD Thesis 

**Evidence for a remote lung response in endotoxin-mediated direct lung injury: a strategic application of microarray technology.**<br>    
**Supervisors:** Dr David Collie & Dr Varrie Ogilvie<br>  
Construction and assay of a suppression-subtraction-hybridisation ovine lung sequence library followed by molecular genomic probing of sheep lung tissue from in vivo models of acute lung injury via novel microarray and qRT-PCR to characterise inflammatory responses in lung segments.

### BSc Honours Thesis

**Design and evaluation of mouse housekeeping genes for oligonucleotide microarrays.**<br>  
**Supervisor:** Dr D Roy<br>  
Analysis of housekeeping genes in mice for the purposes of validating and assuring quality of internally normalised microarray data for mouse models.

## Research skills

**Computational Biology:** RNA-seq, DNA-seq and microarray analysis, R programming, Python scripting, Bioinformatics skills, Machine learning, Image analysis<br>  
**Molecular Biology:** PCR (including qPCR and RT-PCR), molecular cloning, Western blotting, library construction, microarray design, construction and hybridization<br>  
**Cell Biology:** Cell and tissue culture (primary, mammalian and tumour), FACS, siRNA Clinical Research Trial design and statistical analysis training
